clients/test_client/.swagger-codegen-ignore

```diff
@@ -23,3 +23,4 @@
 #!docs/README.md
 
 mix.exs
+config/*
```

Comment:
Nit: Can we standardize on having a trailing newline?
Also for several more files below.

Reply:
Fixed